docs/vector-index/develop/api-reference/index.md

```diff
@@ -0,0 +1,144 @@
+---
+sidebar_position: 1
+sidebar_label: API Reference
+title: MVI API reference
+description: Learn how to interact programmatically with Momento Vector Index API.
+---
+
+import { SdkExampleTabs } from "@site/src/components/SdkExampleTabs";
+// This import is necessary even though it looks like it's un-used; The inject-example-code-snippet
+// plugin will transform instances of SdkExampleTabs to SdkExampleTabsImpl
+import { SdkExampleTabsImpl } from "@site/src/components/SdkExampleTabsImpl";
+
+# Using the Momento Vector Index API
+Momento Vector Index (MVI) is a scalable, developer-friendly vector index service designed for real-time storage and retrieval of vector data for use in AI-powered applications.
+
+## Vector Index methods
+
+### Create Index
+Creates a vector index.
+
+| Name             | Type   | Description                                                                                                                                            |
+|------------------|--------|--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| indexName        | String | Name of the vector index.                                                                                                                              |
+| numDimensions    | String | Number of dimensions per vector.                                                                                                                       |
+| similarityMetric | String | Metric used to quantify the distance between vectors. Can be cosine similarity, inner product, or euclidean similarity. Defaults to cosine similarity. |
+
+
+<details>
+  <summary>Method response object</summary>
+
+* Success
+* AlreadyExists
```

Comment:
Is it AlreadyExists or IndexAlreadyExists?

Reply:
It's AlreadyExists in js and IndexAlreadyExists in python :-/